src/server/scripts/Spells/spell_item.cpp

```diff
@@ -256,9 +256,16 @@ class spell_item_anger_capacitor : public SpellScriptLoader
                 caster->CastSpell(target, spellId, aurEff);
             }
 
+            void OnRemove(AuraEffect const*, AuraEffectHandleModes)
+            {
+            	if (GetTarget())
```

Comment:
I'm not saying that its needed, I'm saying that I dont know if its needed.

Reply:
Well, in [SpellScript](https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/game/Spells/SpellScript.h#L922) check is not needed.